packages/mdc-menu/simple/foundation.js

```diff
@@ -150,109 +168,27 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
   }
 
   /**
-   * Calculates transition delays for individual menu items, so that they fade in one at a time.
-   * @private
+   * @param {!Corner} corner Default anchor corner alignment of top-left menu corner.
    */
-  applyTransitionDelays_() {
-    const {BOTTOM_LEFT, BOTTOM_RIGHT} = MDCSimpleMenuFoundation.cssClasses;
-    const numItems = this.adapter_.getNumberOfItems();
-    const {height} = this.dimensions_;
-    const transitionDuration = MDCSimpleMenuFoundation.numbers.TRANSITION_DURATION_MS / 1000;
-    const start = MDCSimpleMenuFoundation.numbers.TRANSITION_SCALE_ADJUSTMENT_Y;
-
-    for (let index = 0; index < numItems; index++) {
-      const {top: itemTop, height: itemHeight} = this.adapter_.getYParamsForItemAtIndex(index);
-      this.itemHeight_ = itemHeight;
-      let itemDelayFraction = itemTop / height;
-      if (this.adapter_.hasClass(BOTTOM_LEFT) || this.adapter_.hasClass(BOTTOM_RIGHT)) {
-        itemDelayFraction = ((height - itemTop - itemHeight) / height);
-      }
-      const itemDelay = (start + itemDelayFraction * (1 - start)) * transitionDuration;
-      // Use toFixed() here to normalize CSS unit precision across browsers
-      this.adapter_.setTransitionDelayForItemAtIndex(index, `${itemDelay.toFixed(3)}s`);
-    }
+  setAnchorCorner(corner) {
+    this.anchorCorner_ = corner;
   }
 
   /**
-   * Removes transition delays from menu items.
-   * @private
+   * @param {!AnchorMargin} margin 4-plet of margins from anchor.
    */
-  removeTransitionDelays_() {
-    const numItems = this.adapter_.getNumberOfItems();
-    for (let i = 0; i < numItems; i++) {
-      this.adapter_.setTransitionDelayForItemAtIndex(i, null);
+  setAnchorMargin(margin) {
+    if (typeof margin.top == 'number') {
```

Comment:
Nit: triple-equals for comparisons (not technically necessary here, but consistent with other MDC code)

Reply:
done